src/intellij/README.md

```diff
@@ -1,50 +1,57 @@
 # Developing Scala in IntelliJ IDEA
 
-Use the latest IntelliJ release and install the Scala plugin from within the IDE.
+Use the latest IntelliJ release.
+
+Recent releases should come with the Scala plugin preinstalled. (But
+if not, install it before proceeding.)
 
 ## Initial Setup
 
-To create the IntelliJ project files:
+Do not attempt to import our sbt build into IntelliJ; it won't work.
+
+Instead, create IntelliJ project files as follows.
 
   - Run `sbt intellij`
-  - Open `src/intellij/scala.ipr` in IntelliJ
-  - In `File` → `Project Structure` → `Project` → `Project SDK`, create an SDK entry
-    named "1.8" containing the Java 1.8 SDK (1.6 if you're on the Scala the 2.11.x branch)
 
 The project files are created as copies of the `.SAMPLE` files, which are under version
 control. The actual IntelliJ project files are in `.gitignore` so that local changes
 are ignored.
 
-## Dependencies
+Then to start coding:
 
-For every module in the IntelliJ project there is a corresponding `-deps` library, for example `compiler-deps` provides JARs for the compiler codebase.
-The `.jar` files in these `-deps` libraries can be easily kept up-to-date by running `sbt intellij` again.
-This is necessary whenever the dependencies in the sbt build change, for example when the `starr` version is updated.
+  - Open `src/intellij/scala.ipr` in IntelliJ
+  - Wait for the project to index
+  - On the `Build` menu, choose `Build Project`
```

Comment:
Do we know this to be true?  Unless we do, I wouldn't teach this, and rather continue to refer users to running tasks in sbt.

Reply:
Building with IntelliJ, via the `Build Project` menu item, rather than running things in sbt.

Reply:
It's fine either way. There's some text later in the readme that explains that the build products are in the same place either way.

And the previous version also recommended "Build Project", I just moved the recommendation to earlier in the file.

As I understand it anyway, there was never any intention to steer people towards building with sbt as the primary. Building with IntelliJ is fine. Those who are more comfortable with sbt may prefer an sbt-centric workflow. But I don't want to give readers the impression that sbt is actually needed except to run `sbt intellij` at the start.

Reply:
> And the previous version also recommended "Build Project", I just moved the recommendation to earlier in the file.

I missed that - I see it's changed in IntelliJ from "Make Project".

> But I don't want to give readers the impression that sbt is actually needed except to run `sbt intellij` at the start.

I'm fine with the change now, but I wouldn't want the opposite: to give the impression that everything is the same doing it either way - we don't know that and I don't trust that to be true now or in the future.

Reply:
Yeah, perhaps we'll end up refining the wording further.

In any case, I think it's good to have "Build Project" near the top, because doing it at least once verifies that IntelliJ is working properly.